protos/sources/vega/data/v1/spec.proto

```diff
@@ -108,3 +138,13 @@ message Condition {
     OPERATOR_LESS_THAN_OR_EQUAL = 5;
   }
 }
+
+message ExternalDataSpec {
+  DataSpec spec = 1;
+}
+
+// An oracle spec contains an embedded data source spec.
+// Will be generated by the C++ as: OracleSpec_DataSpec_*
```

Comment:
`C++` ?

Reply:
I meant protoc compiler; will correct that, thanks!